src/places.py

```diff
@@ -146,6 +160,28 @@ def _setup(self):
                 self.XDG_TRASH,
             )
 
+        # Bookmarked paths
+
+        try:
```

Comment:
I would much prefer to move all this parsing and bookmark handling logic to a separate / independent model, e.g., say PortfolioBookmarks, aan equivalent to PortfolioDevices class.